infra/terraform/modules/service/batch.tf

```diff
@@ -74,6 +74,16 @@ locals {
     attempt_duration_seconds = job.timeout
     retry_strategy           = local.default_retry_policy
   } }
+
+  schedules = {
+    for job in var.batch.jobs : job.name => {
+      description         = "Schedule for ${job.name}"
+      schedule_expression = job.schedule
+      arn                 = "arn:aws:scheduler:::aws-sdk:batch:submitJob"
+      input               = jsonencode({ "jobName" : job.name, "jobQueue" : module.batch.job_queues.default.arn, "jobDefinition" : module.batch.job_definitions[job.name].arn })
```

Comment:
For job name, we're adding prefix so that it's multi-environment friendly in this module so the `job.name` will not exist. To get the job name, let's use: `module.batch.job_definitions[job.name].name`.

For readability, it may also be worth to update the `description` with either `module.batch.job_definitions[job.name].name` or add an environment in the description somewhere "Schedule for ${var.environment} ${job.name}"

Reply:
In the plan, `job.name` will be `process-queue` but it's actually named `vol-app-[env]-process-queue`